src/engraving/libmscore/cmd.cpp.uncrustify

```diff
@@ -0,0 +1,4438 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
```

Comment:
I guess this file doesn't belong into the PR

Reply:
I'd removed it within minutes!

Reply:
But not pushed it equally fast I guess? Anyway, issue is sorted now.